package/MDAnalysis/core/universe.py

```diff
@@ -317,14 +329,23 @@ class Universe(object):
     .. versionchanged:: 2.0.0
         Universe now can be (un)pickled.
         ``topology`` and ``trajectory`` are reserved upon unpickle.
+
     .. versionchanged:: 2.5.0
         Added fudge_factor and lower_bound parameters for use with
         *guess_bonds*.
+
+    .. versionchanged:: 2.7.0
+        Added :ref:`guess_TopologyAttributes <guess_TopologyAttributes>` API
```

Comment:
```suggestion
        Added :meth:`~MDAnalysis.core.universe.Universe.guess_TopologyAttributes` API
```

Using a reference is really neat and I never thought of doing it this way! It might be clearer though to reference the method directly in this particular context.